### PR TITLE
Publish docs

### PR DESCRIPTION
**Pull request by mdedetrich**
Same implementation that was taken from pekko-http

### CODE REVIEW

File: .github/workflows/publish-nightly.yml [hunk 1]
```diff
@@ -31,3 +31,47 @@ jobs:
...
           NEXUS_PW: ${{ secrets.NEXUS_PW }}
         run: sbt +publish
+
+  publish-docs:
+    name: Publish documentation to nightlies.apache.org
+    runs-on: ubuntu-latest
+    if: github.repository == 'apache/incubator-pekko-projection'
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
```
Comment (pjfanning) [inline .github/workflows/publish-nightly.yml:41]: can we use v3 of these core actions (node.js warnings)?

Comment (mdedetrich) [reply .github/workflows/publish-nightly.yml]: Fixed

Comment (He-Pin) [reply .github/workflows/publish-nightly.yml]: is there a repo we can share this kind of settings cross projects？

**Review by pjfanning — APPROVED** [review]
lgtm

File: .github/workflows/publish-nightly.yml (hunk 1 above)
Comment (mdedetrich) [reply .github/workflows/publish-nightly.yml]: I had an idea to abstract this doc publishing into an sbt plugin, see https://github.com/apache/incubator-pekko/issues/104. This isn't a priority howerver

**Pull request merged**